### PR TITLE
Preview backend

### PR DESCRIPTION
**Pull request by TimYuenior**
Not much to be added to backend integration at this point actually.
Added something in comments for future use.

Also, the user can now scroll through the shotBox in the preview menu with the buttons at the sides

### CODE REVIEW

File: src/main/java/nl/tudelft/contextproject/gui/PreviewController.java [hunk 1]
```diff
@@ -58,13 +62,18 @@
...
     private List<Shot> shots;
     private List<ImageView> views;
+    private Script script;
 
     @FXML private void initialize() {
-        currentSelected = 1;
+        // backend
+//        script = ContextTFP.getScript();
+//        shots = script.getShots();
+//        actualCam = shots.get(0).getCamera();
```
Comment (tkolenbrander) [inline src/main/java/nl/tudelft/contextproject/gui/PreviewController.java:75]: The actualCam should be script.getCurrentShot().getCamera() not shots.get(0).getCamera()

**Comment by tkolenbrander** [comment]
When you fix my comment above, I will approve this pull request. The rest of the code is fine.

File: src/main/java/nl/tudelft/contextproject/gui/PreviewController.java (hunk 1 above)
Comment (TimYuenior) [inline src/main/java/nl/tudelft/contextproject/gui/PreviewController.java:75]: This is just for initialization, so the first shot is taken as an initial camera view.

Comment (tkolenbrander) [inline src/main/java/nl/tudelft/contextproject/gui/PreviewController.java:75]: Ah, that makes sense. In that case I will merge your pull request

**Pull request merged**